Wireframe/css/styles.css

```diff
@@ -0,0 +1,134 @@
+@import url("https://fonts.googleapis.com/css2?family=Karla:ital,wght@0,200;0,300;0,400;0,500;1,200;1,300;1,400;1,500&display=swap");
```

Comment:
I'd import this in the head, by habit. This is a minor point to be honest, but you might find it interesting. Turn on the Performance audit and take a look at what happens when you import css files inside each other.